services/databaseService.js

```diff
@@ -0,0 +1,47 @@
+const { db } = require('../config');
```

Comment:
Is there a reason why we need to separate these functions into another file? I think it should be fine to have these in routes/boxForm.js since that should be the only place we need them, but let me know if there was any reason or standard you're following.

Reply:
I moved them to a different folder because my tech lead Albert from last year's CTC advised us to follow the route-controller-service structure which is essentially separating out the database logic from the routes so I just thought I'd follow that but I can put them all in boxForm.js if you'd like

Reply:
We took a look into the route-controller-service and it looks good to use! Could we rename this file to boxFormService or something more specific to this use case?